custom/icds_reports/static/icds_reports/js/spec/download.directive.spec.js

```diff
@@ -588,7 +588,7 @@ describe('Download Directive', function () {
 
         it('tests that block user does not have access to dashboard usage report', function () {
             var length = controller.indicators.length;
-            assert.equal(10, length);
+            assert.equal(11, length);
```

Comment:
one little thought is that it'd be nice if changing these three values could be done by only changing one value. so something like:

`var numberOfReports = 12`

as a variable scoped to the whole test and then `assert.equal(numberOfReports - 1, length);` etc. (not sure if that's the right variable name as I don't have the full context on these tests)

Reply:
that makes sense to me.